spring-cloud-dataflow-dependencies/pom.xml

```diff
@@ -12,139 +12,62 @@
 	<packaging>pom</packaging>
 	<name>spring-cloud-dataflow-dependencies</name>
 	<description>Spring Cloud Data Flow Dependencies BOM designed to support consumption of Spring Cloud Data Flow from the Spring Initializr.</description>
-	<properties>
-		<spring-analytics.version>1.1.0.M1</spring-analytics.version>
-		<spring-cloud-task.version>1.1.0.BUILD-SNAPSHOT</spring-cloud-task.version>
-		<spring-cloud-commons.version>1.1.3.RELEASE</spring-cloud-commons.version>
-		<spring-cloud-config.version>1.2.0.RELEASE</spring-cloud-config.version>
-		<spring-batch-admin-dependencies.version>1.3.1.RELEASE</spring-batch-admin-dependencies.version>
-		<spring-cloud-dataflow-ui.version>1.1.0.M2</spring-cloud-dataflow-ui.version>
-		<spring-shell.version>1.2.0.RELEASE</spring-shell.version>
-		<spring-session.version>1.2.2.RELEASE</spring-session.version>
-		<spring-cloud-deployer.version>1.1.0.M1</spring-cloud-deployer.version>
-	</properties>
 	<dependencyManagement>
 		<dependencies>
 			<dependency>
 				<groupId>org.springframework.cloud</groupId>
-				<artifactId>spring-cloud-task-dependencies</artifactId>
-				<version>${spring-cloud-task.version}</version>
-				<type>pom</type>
-				<scope>import</scope>
-			</dependency>
-			<dependency>
-				<groupId>org.springframework.cloud</groupId>
-				<artifactId>spring-cloud-commons-dependencies</artifactId>
-				<version>${spring-cloud-commons.version}</version>
-				<type>pom</type>
-				<scope>import</scope>
-			</dependency>
-			<dependency>
-				<groupId>org.springframework.cloud</groupId>
-				<artifactId>spring-cloud-config-dependencies</artifactId>
-				<version>${spring-cloud-config.version}</version>
-				<type>pom</type>
-				<scope>import</scope>
-			</dependency>
-			<dependency>
-				<groupId>org.springframework.batch</groupId>
-				<artifactId>spring-batch-admin-manager</artifactId>
-				<version>${spring-batch-admin-dependencies.version}</version>
-			</dependency>
-			<dependency>
-				<groupId>org.springframework.analytics</groupId>
-				<artifactId>spring-analytics</artifactId>
-				<version>${spring-analytics.version}</version>
-			</dependency>
-			<dependency>
-				<groupId>org.springframework.cloud</groupId>
 				<artifactId>spring-cloud-dataflow-shell</artifactId>
-				<version>1.1.0.BUILD-SNAPSHOT</version>
+				<version>${project.version}</version>
```

Comment:
Any reason why you don't stick with the hard-coded value? This is replaced automatically on release and is clearer when reading the pom IMO